core/arch/arm/include/sm/sm.h

```diff
@@ -37,6 +37,10 @@ struct sm_unbanked_regs {
 #ifdef CFG_SM_NO_CYCLE_COUNTING
 	uint32_t pmcr;
 #endif
+#ifdef CFG_TA_FTRACE_SUPPORT
+	uint32_t cntkctl;
+	uint32_t pad;
```

Comment:
Why is `pad` needed?

Reply:
For 8 byte alignment of structs. Have a look at compile time asserts here: https://github.com/OP-TEE/optee_os/blob/14d6d42b476edcd83181b357ffd70222a70cdc36/core/arch/arm/sm/sm.c#L20

Reply:
OK.